### DISCUSSION TITLE
PNP - Web part not available

### DISCUSSION THREAD

**Discussion by CLey7** [microsoft-search/pnp-modern-search#2186] [Q&A]
Hi there, 
i am just after some guidance on PNP Web Search as i am experiencing a niggly issue setting it up.

i got the SPFK File from github which was PNP V4 Search tool, added the app to my 365 app admin centre. Then I gave all the permisisons on API to use the app. I then added the PNP to a certain site and then when I click the icon called Web part seach and then press configure, I do not see the option for SHarepoint Search or Microsoft Search Data soruces. I just get a message on the right on sharepoint which says: 

**PnP - Search Results    No settings to change for this web part.**

I was wondering if someone could guide me in case i have done soemthing wrong or it is not as simple as the steps I have done?

thanks

**Comment by FranckyC** [comment]
Hi @CLey7 , what version are you using? Sometimes it is just a UI bug. Try to add a default SharePoint Web Part on the page (like the button WebPart), edit the settings to open the property pane and then click on the content of the PnP WebPart, the configuration in the edit panel should change automatically.

**Comment by nicolesherman00** [comment]
I too am getting this issue.. But mine is just a white panel instead of even telling me there is no setting to change. 
I tried your resolution above @FranckyC but when I click on the PNP webpart, it just continues to show the settings for the OOTB webpart.

**Reply by wobba** [reply]
Not sure what causes this, but can be related to caching and CDN (if that is enabled). You can try to upload the sppkg again over the existing one and retry. Or clear your cache and retest.

**Reply by nicolesherman00** [reply]
Tried both and unfortunately they did not resolve the issue.

I tried copying the page and testing and still was unable to go into the settings.

I also tried creating a fresh page on the same site, and there was settings but only 2 pages of settings
![image](https://user-images.githubusercontent.com/22782498/191076044-f4137854-0c3c-40bd-9ab0-3235a82c6288.png)

When checking my other site collections that have a PNP search web part they work fine, it seems to just be this one site collection that is affected. 

My sppkg is in the global apps so it updates it everywhere I assume....

**Reply by wobba** [reply]
Global deployment or adding them to the site? If the latter you could try to remove and re-add. Either way, seems a SPFx bundle/manifest loading issue, not web part related.
